docs/docs/references/preview-projects.mdx

```diff
@@ -0,0 +1,173 @@
+
+### Lightdash CLI Tool (Local)
+
+For preview projects created from the CLI, all project connection settings come from your **local** `profiles.yml` file. 
+
+That means that we will use the same default connection settings that you're using locally for development, unless you specify different options in your CLI commands. 
+
+For example:
+- By default, `lightdash preview` will use the `profiles.yml` file located at `~/.dbt/profiles.yml`. If you want to use a different `profiles.yml` file, you can specify this using the option `lightdash preview --profiles-dir path/to/directory`.  
+- `lightdash preview` will use whatever default target that is set in your `profiles.yml` file. The database connection settings (schema, project, dataset, etc.) from this target will be used as the database connection settings for your preview project. You can override the target used to create your preview project using the flag `lightdash preview --target my-target-name`  
+
+### Lightdash App
+
+For preview projects created in the Lightdash app, all project connection settings are copied directly from the project the preview is built from. 
+
+For example:
+- I have a project called `Lightdash Analytics` and I create a preview of that project called `Preview Lightdash Analytics`
+- `Preview Lightdash Analytics` will have the same project connection settings (warehouse connection, dbt project connection, etc.) as `Lightdash Analytics`.  
+
+The project connection settings cannot be changed in a preview project.
+
+### Pull Requests (GitHub Actions) 
+
+For preview projects created from GitHub actions, the project connection settings come from the `profiles.yml` file that you've specified to use in your GitHub action. 
+
+If you haven't changed anything from our default [GitHub action script](https://github.com/lightdash/cli-actions/blob/main/start-preview.yml), then the connection settings used for your preview project should be whatever you've specified in your [`DBT_PROFILES` secret](/guides/cli/how-to-use-lightdash-preview#dbt_profiles).
+
+You have three options for setting up project connections in your GitHub Actions:
+1. set up a single project connection that's used in every pull request. E.g. always use the `prod` target. You can [follow the instructions here](/guides/cli/how-to-use-lightdash-preview#dbt_profiles) for your data warehouse
+2. if you're using dbt Cloud, you can [use the dbt Cloud schema](/guides/cli/how-to-use-lightdash-preview#how-to-use-the-dbt-cloud-schema-in-your-preview-project) to manage the connection used in the pull request.
+3. [create a project connection per developer in GitHub](/guides/cli/how-to-use-lightdash-preview#how-to-use-the-developer-credentials-in-your-preview-project) (similar to `dev` profiles in your dbt project) so the connection changes depending on who opened the pull request. Tis is useful if you use different development schemas for team members developing in dbt.
```

Comment:
typo: `Tis` should be `This`

Just to be annoying: these sentences should start with capital letters I think